src/EnergyPlus/RoomAirModelManager.cc

```diff
@@ -133,43 +125,46 @@ namespace RoomAirModelManager {
             SharedDVCVUFDataInit(state, ZoneNum);
         }
 
-        {
-            auto const SELECT_CASE_var(state.dataRoomAirMod->AirModel(ZoneNum).AirModelType);
-
-            if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::UserDefined) {
-
-                ManageUserDefinedPatterns(state, ZoneNum);
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::Mixing) { // Mixing air model
-                                                                                    // do nothing
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::Mundt) { // Mundt air model
-                // simulate room airflow using Mundt model
-                ManageMundtModel(state, ZoneNum);
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::UCSDDV) { // UCDV Displacement Ventilation model
-                // simulate room airflow using UCSDDV model
-                ManageUCSDDVModel(state, ZoneNum);
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::UCSDCV) { // UCSD Cross Ventilation model
-                // simulate room airflow using UCSDDV model
-                ManageUCSDCVModel(state, ZoneNum);
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::UCSDUFI) { // UCSD UFAD interior zone model
-                // simulate room airflow using the UCSDUFI model
-                ManageUCSDUFModels(state, ZoneNum, DataRoomAirModel::RoomAirModel::UCSDUFI);
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::UCSDUFE) { // UCSD UFAD exterior zone model
-                // simulate room airflow using the UCSDUFE model
-                ManageUCSDUFModels(state, ZoneNum, DataRoomAirModel::RoomAirModel::UCSDUFE);
-
-            } else if (SELECT_CASE_var == DataRoomAirModel::RoomAirModel::AirflowNetwork) { // RoomAirflowNetwork zone model
-                // simulate room airflow using the AirflowNetwork - based model
-                SimRoomAirModelAirflowNetwork(state, ZoneNum);
-
-            } else { // mixing air model
-                     // do nothing
-            }
+        switch (state.dataRoomAirMod->AirModel(ZoneNum).AirModelType) {
```

Comment:
The SCV was already enum based, just needed conversion to a proper switch block.